web/virtual_lab/views.py

```
@@ -0,0 +1,45 @@
+# web/virtual_lab/views.py
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)